TACC/Core-CMS
# fix(css): bump text editor css core-styles to v2.9

## Overview

Use [Core-Styles WYSIWIG stylesheet at v2.9.0](https://github.com/TACC/Core-Styles/blob/v2.9.0/dist/core-styles.wysiwyg.css) not v2.6.1.

## Related

Skipped.

## Changes

- **changed** text editor stylesheet core-styles version

## Testing

0. Have Dev Tools "network" tab open.
1. Open CMS Text plugin instance editor.
2. Verify `contents.css` loads with v2.9.0 styles.

## UI

…
